tests/FlatBuffers.Test/NetTest.sh

```diff
@@ -1,23 +1,41 @@
 #!/bin/sh
 
+# Restore nuget packages
+mkdir dotnet_tmp
+curl -OL https://dot.net/v1/dotnet-install.sh
+chmod +x dotnet-install.sh
+./dotnet-install.sh --version 3.1.101 --install-dir dotnet_tmp
+dotnet_tmp/dotnet new sln
+dotnet_tmp/dotnet sln add FlatBuffers.Test.csproj
+curl -OL https://dist.nuget.org/win-x86-commandline/v5.4.0/nuget.exe
+mono nuget.exe restore
+
+# Copy Test Files
+cp ../monsterdata_test.mon Resources/
+cp ../monsterdata_test.json Resources/
+
 # Testing C# on Linux using Mono.
 
-mcs -debug -out:./fbnettest.exe \
-  ../../net/FlatBuffers/*.cs ../MyGame/Example/*.cs ../MyGame/Example2/*.cs ../MyGame/*.cs ../union_vector/*.cs \
-  FlatBuffersTestClassAttribute.cs FlatBuffersTestMethodAttribute.cs Assert.cs FlatBuffersExampleTests.cs Program.cs ByteBufferTests.cs FlatBufferBuilderTests.cs FlatBuffersFuzzTests.cs FuzzTestData.cs Lcg.cs TestTable.cs
-mono --debug ./fbnettest.exe
-rm fbnettest.exe
+msbuild -property:Configuration=Release,OutputPath=tempcs -verbosity:minimal FlatBuffers.Test.csproj
+mono tempcs/FlatBuffers.Test.exe
```

Comment:
To share nuget package settings,  I replaced `mcs` to `msbuild`.
mono has `msbuild` and able to shared windows `csproj` build settings.

```
// windows test
msbuild.exe /property:Configuration=Release;OutputPath=tempcs /verbosity:minimal FlatBuffers.Test.csproj
tempcs\\FlatBuffers.Test.exe
```